builtin/game/chatcommands.lua

```diff
@@ -679,19 +679,36 @@ core.register_chatcommand("status", {
 })
 
 core.register_chatcommand("time", {
-	params = "<0...24000>",
+	params = "<0..23>:<0..59> | <0..24000>",
 	description = "set time of day",
 	privs = {settime=true},
 	func = function(name, param)
 		if param == "" then
-			return false, "Missing time."
-		end
-		local newtime = tonumber(param)
-		if newtime == nil then
-			return false, "Invalid time."
-		end
-		core.set_timeofday((newtime % 24000) / 24000)
-		core.log("action", name .. " sets time " .. newtime)
+			local current_time = math.floor(core.get_timeofday() * 1440)
+			local minutes = current_time % 60
+			local hour = (current_time - minutes) / 60
+			return true, ("Current time is %2d:%2d"):format(hour , minutes)
```

Comment:
This prints times like "6: 6". Better is "6:06", with "Current time is %2d:%02d".